views/index.jade

```diff
@@ -7,7 +7,9 @@ block content
         // DISPLAY LOGIN OR REGISTER WHEN NOT LOGGED IN
         if (!currentUser)
             //p.itinerary-add-message Login or register
-        h1.myclaas Itinerary List
+        center
+          h1.myclaas List of Awesome Activity Planners
```

Comment:
I don't think the class 'myclaas' is in the stylesheet; I also couldn't find 'myclass', so I am wondering if we need to label this div with that class name. 
While I do think that the content of this header aligns with the name of our app, the actual name of our app is not correct in the top left of the nav bar.

Reply:
Yes. I added a quick fix and created the myclaas in the stylesheet,  So I can align the center